unit/Makefile

```diff
@@ -168,6 +168,7 @@ BMC_DEPS =../src/cbmc/c_test_input_generator$(OBJEXT) \
           ../src/goto-cc/armcc_cmdline$(OBJEXT) \
           ../src/goto-cc/goto_cc_cmdline$(OBJEXT) \
           ../src/goto-instrument/source_lines$(OBJEXT) \
+          ../src/goto-instrument/abstraction_spect$(OBJEXT) \
```

Comment:
??? I think the unit tests are missing?  Also `rr_abstraction_spect` or `replication_reducting_abstraction_spect`?

Reply:
This dependency is no longer needed. Changed  to rr_abstraction_spect.